modules/virt-copying-the-ssh-command.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+
+:_content-type: PROCEDURE
+[id="virt-copying-the-ssh-command_{context}"]
+= Copying the SSH Command from the web console
+
+Copy the SSH command of your virtual machine (VM) from the web console for use in the OpenShift CLI (`oc`).
+
+.Prerequisites
+* A configured virtual machine.
+
+.Procedure
+
+. In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
```

Comment:
@sjhala-ccs @sabrinajess 
Just wondering, why is it called the OpenShift Virtualization console instead of the OpenShift Container Platform console? I never understood the logic. Normally, I think of a "whatever" console as a standalone web console, and not as a menu item in a larger console.

Reply:
@apinnick It should be OpenShift Container Platform web console. Good catch.